### DISCUSSION TITLE
[THEME] Bing Daily theme

### DISCUSSION THREAD

**Discussion by Nriver** [TriliumNext/Trilium#6067] [Show and tell]
https://github.com/Nriver/bing-daily-theme

Hi, I've just created a new theme to share with you all.

It's a theme that automatically updates with the Bing daily wallpaper. Give it a try!

![shot1](https://raw.githubusercontent.com/Nriver/bing-daily-theme/main/docs/screenshot%201.webp)

**Comment by CobriMediaJulien** [comment]
Cool, thank you! I will try it.
